src/cms/actions/create-posts-page.action.ts

```diff
@@ -20,18 +21,16 @@ export const createPostsPage = async (
       data,
     );
 
-    switch (response.status) {
-      case 201:
-        return response.data;
+    return response.data;
+  } catch (error: SherlError | Error | any) {
+    switch ((error as SherlError).data?.status) {
       case 403:
         throw errorFactory.create(CmsErr.CREATE_CMS_POSTS_FAILED_CMS_FORBIDDEN);
```

Comment:
```suggestion
        throw errorFactory.create(CmsErr.CREATE_CMS_POSTS_CMS_FORBIDDEN);
```

On a enlevé les FAILED dans d'autres domaines, si on peut être cohérent.